cmd/web-handlers.go

```diff
@@ -692,6 +694,7 @@ next:
 					ObjectName:      objectName,
 					Claims:          claims.Map(),
 				}) {
+					r.Header.Add("X-Amz-Bypass-Governance-Retention", "true")
```

Comment:
We shouldn't add extra headers in this manner @ebozduman

Reply:
OK. Then what is the right way to have that header in?
Is it supposed to happen at the front end level?

Reply:
You can never do that, browser request should add it if necessary @ebozduman otherwise you let the request fail.